src/aleph/sdk/query/responses.py

```diff
@@ -77,5 +77,5 @@ class MessagesResponse(PaginationResponse):
 class PriceResponse(BaseModel):
     """Response from an aleph.im node API on the path /api/v0/price/{item_hash}"""
 
-    required_tokens: float
+    required_tokens: str
```

Comment:
`pyaleph` should expose the information as a string, and every client should handle it given its own properties (ex: Python = Decimal)

```suggestion
    required_tokens: Decimal
```

Reply:
![image](https://github.com/user-attachments/assets/d1cfb6a5-d206-421a-8dd3-635fa52d017c)

Reply:
We agreed in the meeting of today to use the integer version as a string, a field containing the number of decimals of the token, and the (low-precision) float representation of the value.